initial/src/main/java/com/example/springboot/HelloController.java

```diff
@@ -1,14 +1,40 @@
 package com.example.springboot;
 
+import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.bind.annotation.RequestMapping;
+import java.util.Random;
 
 @RestController
 public class HelloController {
 
 	@RequestMapping("/")
-	public String index() {
-		return "Hello Ethan Wei, welcome to Spring Boot!";
+	//create algorithm under controller to send the calculated value to the server
+	public String index(String[] args) {
+		Random rand = new Random();
+		int upperbound = 25;
```

Comment:
Can you move this variable to class level and `private static final` should be used to decorate it.